static/src/js/actions/portals.js

```diff
@@ -14,12 +15,16 @@ export const addPortal = (payload) => ({
  * Loads the portal config into the Redux Store.
  * @param {String} portalId Portal Name, must match the directory name in which the config is stored.
  */
-export const loadPortalConfig = (portalId) => (dispatch) => {
-  if (!portalId) return
+export const loadPortalConfig = (
+  portalId = getApplicationConfig().defaultPortal
```

Comment:
Does this need to be defaulted?

Reply:
In my branch It isn't calling the function without args but, I handled the static.webpack.config, differently from the branch I'm trying to merge into. I think that even that has now change because we are going to be reading the files directly.

Reply:
I'm going to remove it from this branch we can return a default if we need to but, I think there were changes to actions/portals.js regardless in the working dir of CMR-3546